CMakeLists.txt

```diff
@@ -29,7 +40,12 @@ file(GLOB_RECURSE sources CONFIGURE_DEPENDS "src/*.c" "src/*.cpp" "src/*.h" "src
 add_library(OpenHotfixLoader SHARED ${sources} "${CMAKE_CURRENT_BINARY_DIR}/versioninfo.rc")
 target_include_directories(OpenHotfixLoader PRIVATE "${PROJECT_BINARY_DIR}/inc" "src")
 
-target_link_libraries(OpenHotfixLoader PUBLIC minhook shlwapi cpr::cpr plog)
+if (DEFINED CMAKE_MINGW32)
+   target_link_libraries(OpenHotfixLoader PUBLIC minhook libshlwapi.a cpr::cpr plog)
+else()
+   target_link_libraries(OpenHotfixLoader PUBLIC minhook shlwapi cpr::cpr plog)
+endif()
```

Comment:
Probably a bit cleaner to have one link at the top with the constant libs,, and then only shlwapi in the if

Reply:
I didn't make a list :/ I just made some vars. Why do libs have 2 names (lib name + link name).